diff --git a/term-utils/agetty.c b/term-utils/agetty.c
index cecde2c..94cdd25 100644
--- a/term-utils/agetty.c
+++ b/term-utils/agetty.c
@@ -520,7 +520,7 @@ int main(int argc, char **argv)
 		reset_vc(&options, &termios, 1);
 
 	/* Now the newline character should be properly written. */
-	write_all(STDOUT_FILENO, "\r\n", 2);
+	write_all(STDOUT_FILENO, "\r", 2);
 
 	sigaction(SIGQUIT, &sa_quit, NULL);
 	sigaction(SIGINT, &sa_int, NULL);
